Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.6-alpine
+
+WORKDIR "/root"
+COPY bowser /root/bowser
+COPY scripts /root/scripts
+COPY setup.py /root/setup.py
+RUN pip install -e .
+
+ENTRYPOINT ["bowser-bot"]
```

Comment:
You should be using `CMD` instead of `ENTRYPOINT` unless your `ENTRYPOINT` explicitly handles executing the `CMD` after it does things. This is commonly done to do pre-setup work so you might install the package from source, or you might setup some storage.